conf/config.level0.neon

```diff
@@ -30,6 +30,7 @@ rules:
 	- PHPStan\Rules\Classes\ExistingClassInTraitUseRule
 	- PHPStan\Rules\Classes\InstantiationRule
 	- PHPStan\Rules\Classes\InvalidPromotedPropertiesRule
+	#- PHPStan\Rules\Classes\LocalTypeAliasesRule
```

Comment:
Why is it commented out? Although new rules shouldn't be added in minor versions, this is fine as it's for a new feature - no existing code should be affected.

Reply:
Oh, that's a mistake, thanks for catching it!